### ISSUE TITLE
Failed to find bundle version when running with Docker

### ISSUE THREAD

**Issue by frasalvi** [empiricaly/empirica#498]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

I am trying to deploy my experiment with v1.9.3, but I'm getting a `failed to find bundle version` error when I try to run `empirica serve` from a Docker container built from `ghcr.io/empiricaly/empirica:build-v1.9.3`; this used to work until the previous version. I managed to recreate a minimum working example as detailed below.

### Steps To Reproduce

1. Create a new experiment, i.e. `empirica create myproject`
2. Run `empirica bundle`
3. Create a Dockerfile as:

```
FROM ghcr.io/empiricaly/empirica:build-v1.9.3

WORKDIR /
COPY myproject.tar.zst /app/myproject.tar.zst

EXPOSE 3000

CMD ["empirica", "serve", "/app/myproject.tar.zst"]
```

4. Build the image: `docker build -t myproject .`
6. Run the image: `docker run -it --rm myproject`

### Empirica Version

```text
Version: v1.9.3
SHA:     0721d5b
Build:   176
Branch:  main
Time:    2024-01-21T08:00:13Z

Client:  1.9.3
Server:  1.9.3
```

### What OS are you seeing the problem on?

Linux

### What browser are you seeing the problem on?

Does not apply

### Relevant log output

```shell
% Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 10930  100 10930    0     0  13123      0 --:--:-- --:--:-- --:--:-- 13121
  Installing latest version of Volta (1.1.1)
    Checking for existing Volta installation
    Fetching archive for Linux, version 1.1.1
######################################################################## 100.0%
    Creating directory layout
  Extracting Volta binaries and launchers
    Finished installation. No changes were made to user profile settings.
success: installed and set node@20.11.0 (with npm@10.2.4) as default

01:03:13.589 FTL failed to find bundle version error="get latest installed version: no installed versions"
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by npaton** [comment]
I tried to add a quick fix for this in the latest release (1.9.4), though it is not tested yet. Can you try on your side and see if it's working for you. Thanks!

**Comment by frasalvi** [comment]
That seems to have solved it, thanks!
